queue/driver.go

```diff
@@ -25,6 +25,8 @@ type remoteQueueDriver interface {
 	GetAllAttempts(ctx context.Context, id string) ([]amboy.Job, error)
 	// Put inserts a new job in the backing storage.
 	Put(context.Context, amboy.Job) error
+	// PutMany inserts new jobs in the backing storage.
+	PutMany(context.Context, []amboy.Job) error
```

Comment:
Comment for how it handles multiple errors across different job insertions.

Reply:
done